libs/utils/utils.js

```diff
@@ -1225,10 +1225,10 @@ export function partition(arr, fn) {
   );
 }
 
-const preloadBlockResources = (blocks = []) => blocks.map((block) => {
+export const preloadBlockResources = (blocks = [], marqueesOnly = true) => blocks.map((block) => {
   if (block.classList.contains('hide-block')) return null;
   const { blockPath, hasStyles, name } = getBlockData(block);
-  if (['marquee', 'hero-marquee'].includes(name)) {
+  if (['marquee', 'hero-marquee'].includes(name) || !marqueesOnly) {
```

Comment:
Why would you modify this function? You can't be sure the blocks you preload from within the MEP scripts really use decorate. 

The idea is to have the small tradeoff of 2-3 extra lines for a benefit in most cases (marquee/hero-marquee) which are very common LCP elements.